src/etc/test-float-parse/runtests.py

```diff
@@ -195,9 +195,8 @@ def main():
     global MAILBOX
     tests = [os.path.splitext(f)[0] for f in glob('*.rs')
                                     if not f.startswith('_')]
-    listed = sys.argv[1:]
-    if listed:
-        tests = [test for test in tests if test in listed]
+    args = sys.argv[1:]
+    tests = [test for test in tests if test in args]
```

Comment:
This changes the no-arg behavior, previously it included everyhting now it will include nothing. Was this intentional?

Reply:
Yeah, that's right. Will prepare another PR.